docs/development.md

```diff
@@ -0,0 +1,26 @@
+# Development Notes
+
+## Arrow Functions
+
+This projet heavily prefers the use of the `function` keyword over the use of 
+[arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) 
+in most cases.
+Arrow functions may be used for one line callbacks within methods such as `Array.map()` and `Array.filter()`.
```

Comment:
"within methods such as `Array.map()` and `Array.filter()`."
This sell seems too specific.
It's fine to give an example, but your exact wording here sounds like you are limiting the scope to only a specific class of callbacks
(instead of all callbacks (as long as they can meet the below requirements)).
Often keeping things simple is best.
Something like:
"Arrow functions may only be used for one line callbacks."